### PR TITLE
Bring back the sensitive data filtering feature

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

**Review by gadotroee — COMMENTED** [review]
Looks very nice (not that big change).
Just to be sure I understand correctly `sensitive_data_cleaner.go` is a copied functionality (that probably changed a little bit because of the new format) of the logic we had before, right?

File: tap/passive_tapper.go [hunk 1]
```diff
@@ -96,6 +96,8 @@ var ownIps []string             // global
 var hostMode bool               // global
 var extensions []*api.Extension // global
 
+var filteringOptions *api.TrafficFilteringOptions
```
Comment (gadotroee) [inline tap/passive_tapper.go:99]: add //global comment (to be like others..)

**Comment by mertyildiran** [comment]
> Looks very nice (not that big change).
> Just to be sure I understand correctly `sensitive_data_cleaner.go` is a copied functionality (that probably changed a little bit because of the new format) of the logic we had before, right?

@gadotroee yes, it's providing the same functionality of https://github.com/up9inc/mizu/blob/17fa163ee310d94c3c6efea0509b014312290842/agent/pkg/sensitiveDataFiltering/messageSensitiveDataCleaner.go

File: tap/passive_tapper.go (hunk 1 above)
Comment (mertyildiran) [reply tap/passive_tapper.go]: @gadotroee added with https://github.com/up9inc/mizu/commit/73ed5b9a7776cfe87f77c9fe86159e79e57f0339

Comment (gadotroee) [reply tap/passive_tapper.go]: 👍

**Review by gadotroee — APPROVED** [review]
Looking good

**Pull request merged**